scripting/get5/recording.sp

```diff
@@ -94,9 +94,63 @@ public Action Timer_FireStopRecordingEvent(Handle timer, DataPack pack) {
   Call_PushCell(event);
   Call_Finish();
   EventLogger_LogAndDeleteEvent(event);
+
+  UploadDemoToServer(demoFileName);
   return Plugin_Handled;
 }
 
+void UploadDemoToServer(const char[] demoFileName) {
+  bool steamWorksAvaliable = LibraryExists("SteamWorks");
+
+  if (!steamWorksAvaliable) {
+    LogDebug("Cannot upload demoes to a web server without the SteamWorks extension running.");
```

Comment:
Also this needs to go **after** the "will not upload because URL is not set, otherwise everyone gets this error all the time if they don't have SteamWorks.